Create ui-c8y-1023-52-0-add-set-to-current-time-button-to-history-11214-graft-release.md (#4365)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-52-0-add-set-to-current-time-button-to-history-11214-graft-release.md

---
date: ""

> title: "Now" button in history mode

Reviewer: ```suggestion
title: Now button added in global context history mode configuration
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66067
version: 1023.52.0
---

> In the global context history mode configuration, you can now click the "Now" button next to the "To" date field to instantly set the end date to the current time. This provides a convenient shortcut to view the most recent data without manually adjusting the date picker.

Reviewer: ```suggestion
In the global context history mode configuration, a **Now** button has been added next to the **To** date field to instantly set the end date to the current time. This provides a convenient shortcut to view the most recent data without manually adjusting the date picker.
```
